openmoc/compatible/opencg_compatible.py

```diff
@@ -387,7 +405,8 @@ def get_compatible_opencg_cells(opencg_cell, opencg_surface, halfspace):
   compatible_cells = list()
 
   # SquarePrism Surfaces
-  if opencg_surface.type == 'z-squareprism':
+  if opencg_surface.type in ['x-squareprism', 'y-squareprism', 
+                             'z-squareprism']:
 
```

Comment:
This is more of a personal preference, but you could consider adding some ascii art to describe what the 8 cells correspond to:

    # If Cell is outside the SquarePrism (positive halfspace), add "outside" of Surface halfspaces. 
    # Since OpenMOC does not have a SquarePrism Surface, individual Cells are created 
    # for the 8 Cells that make up the outer region of a SquarePrism.
    #                 |                    |
    #           0     |        1           |    2
    #           ______|____________________|______
    #                 |     SquarePrism    |
    #           7     |   (-)  halfspace   |    3
    #           ______|____________________|______
    #                 |                    |  
    #           6     |        5           |    4
    #                 |                    |